sdk/dashboard/Azure.ResourceManager.Dashboard/api/Azure.ResourceManager.Dashboard.netstandard2.0.cs

```diff
@@ -0,0 +1,146 @@
+        public virtual Azure.Response<Azure.ResourceManager.Dashboard.GrafanaResource> Get(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Dashboard.GrafanaResource>> GetAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.Dashboard.GrafanaResource> RemoveTag(string key, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Dashboard.GrafanaResource>> RemoveTagAsync(string key, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.Dashboard.GrafanaResource> SetTags(System.Collections.Generic.IDictionary<string, string> tags, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Dashboard.GrafanaResource>> SetTagsAsync(System.Collections.Generic.IDictionary<string, string> tags, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.Dashboard.GrafanaResource> Update(Azure.ResourceManager.Dashboard.Models.PatchableGrafanaResourceData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Dashboard.GrafanaResource>> UpdateAsync(Azure.ResourceManager.Dashboard.Models.PatchableGrafanaResourceData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class GrafanaResourceCollection : Azure.ResourceManager.Core.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.Dashboard.GrafanaResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.Dashboard.GrafanaResource>, System.Collections.IEnumerable
+    {
+        protected GrafanaResourceCollection() { }
+        public virtual Azure.ResourceManager.ArmOperation<Azure.ResourceManager.Dashboard.GrafanaResource> CreateOrUpdate(bool waitForCompletion, string workspaceName, Azure.ResourceManager.Dashboard.GrafanaResourceData body = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
It is hard to believe that body of create could be optional, could you please change this in directive?

Reply:
I will ask service to make the change from swagger

Reply:
The problem is not that bad, because we have sorted the parameters by putting body parameter right after other required parameters: https://github.com/Azure/autorest.csharp/pull/1894

So:
- If the body parameter is optional, it will be the first optional parameter
- If the body parameter is required, it will be the last required parameter
- If service team later on fixes the wrong definition, it is likely will not break customers codes, since no body will actually pass in null for body parameter.